pubsub/redis.go

```diff
@@ -65,10 +75,79 @@ func (s *RedisSubscriber) Start() error {
 }
 
 func (s *RedisSubscriber) listen() error {
-	c, err := redis.DialURL(s.url)
 
-	if err != nil {
-		return err
+	var c redis.Conn
+	var err error
+
+	if s.sentinelsEnabled {
+
+		s.log.Debugf("Redis sentinel enabled")
+		s.log.Debugf("Redis sentinel parameters:  sentinels: %s,  masterName: %s", s.sentinels, s.masterName)
+		sentinels := strings.Split(s.sentinels, ",")
+		sntnl := &sentinel.Sentinel{
```

Comment:
I think, we can build a sentinel once (in `Start()` function), not for every connection attempt (the `listen` is called for every reconnection).